core/src/test/scala/org/apache/spark/metrics/sink/PrometheusServletSuite.scala

```diff
@@ -63,7 +63,7 @@ class PrometheusServletSuite extends SparkFunSuite with PrivateMethodTester {
     val key = "local-1592132938718.driver.LiveListenerBus." +
       "listenerProcessingTime.org.apache.spark.HeartbeatReceiver"
     val sink = createPrometheusServlet()
-    val suffix = sink invokePrivate PrivateMethod[String]('normalizeKey)(key)
+    val suffix = sink invokePrivate PrivateMethod[String](Symbol("normalizeKey"))(key)
```

Comment:
will this be broken again if Scala removes symbol completely?

Reply:
OK seems they won't. Then I'm OK with it.

Reply:
Seems so but I don't know they have a plan to remove `Symbol` completely.
At least Scala 3 seems to have `Symbol` itself.
https://scalacenter.github.io/scala-3-migration-guide/docs/incompatibilities/dropped-features.html

Reply:
Quoting the sentence in the doc:

> Although scala.Symbol is useful for migration, beware that it is deprecated and that it will be removed from the scala-library. You are recommended, as a second step, to replace them with plain string literals "xwy" or a dedicated class.

unless the sentence is incorrect I would consider it as targeted as future removal.

Reply:
I missed. Scala 3 seems to deprecate `Symbol`.